src/svg.ts

```diff
@@ -0,0 +1,276 @@
+import { Button, ButtonType, DigitGroup, Frame, GameConfig } from "./@types";
```

Comment:
With the exception of the exported functions, this entire file could be migrated to a SVG build script.